sdk/v1/client_test.go

```diff
@@ -190,25 +190,120 @@ func TestClient_PostJob(t *testing.T) {
 		})
 	})
 
-	Convey("given a 409 response", t, func() {
+	Convey("given a 404 response", t, func() {
 		httpClient := newMockHTTPClient(&http.Response{StatusCode: http.StatusConflict}, nil)
 		searchReindexClient := newSearchReindexClient(t, httpClient)
 
-		Convey("when search-reindexClient.PostJob is called", func() {
+		Convey("When search-reindexClient.PostJob is called", func() {
 			job, err := searchReindexClient.PostJob(ctx, client.Headers{})
 			So(err, ShouldNotBeNil)
 			So(err.Error(), ShouldEqual, "failed as unexpected code from search reindex api: 409")
 			So(apiError.ErrorStatus(err), ShouldEqual, http.StatusConflict)
 
-			Convey("then the expected empty job is returned", func() {
+			Convey("Then the expected empty job is returned", func() {
 				So(job, ShouldResemble, models.Job{})
 			})
 
-			Convey("and client.Do should be called once with the expected parameters", func() {
+			Convey("And client.Do should be called once with the expected parameters", func() {
 				doCalls := httpClient.DoCalls()
 				So(doCalls, ShouldHaveLength, 1)
 				So(doCalls[0].Req.URL.Path, ShouldEqual, path)
 			})
 		})
 	})
 }
+
+func TestClient_PostTasksCount(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	testJobID := "883c81fd-726d-4ea3-9db8-7e7c781a01cc"
+	pathToCheck := "v1/jobs/883c81fd-726d-4ea3-9db8-7e7c781a01cc/tasks"
+
+	mockTaskToCreate := `{"task_name":"zebedee","number_of_documents": "10"}`
```

Comment:
This will also need to be a TaskToCreate:
```suggestion
		mockTaskToCreate := models.TaskToCreate{
		TaskName:          "zebedee",
		NumberOfDocuments: 10,
	}
```

Reply:
nope. again this is a representation of byte slice.